### ISSUE TITLE
Deprecate existing Feature functionality from Stories & Events

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#13692]

**Comment by FranECross** [comment]
@jilladams Do you have info on what this ticket is for? 😁TIA!

**Comment by jilladams** [comment]
Yep, described in the parent epic here: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/1318

Blocked on our ability to prioritze a new featuring mechanism for Stories.
